src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -748,6 +751,8 @@
     text: az openshift create -g MyResourceGroup -n MyManagedCluster --aad-client-app-id {APP_ID} --aad-client-app-secret {APP_SECRET} --aad-tenant-id {TENANT_ID} --compute-count 5
   - name: Create an Openshift cluster using a custom vnet
     text: az openshift create -g MyResourceGroup -n MyManagedCluster --vnet-peer "/subscriptions/0000000-0000-0000-0000-000000000000/resourceGroups/openshift-vnet/providers/Microsoft.Network/virtualNetworks/test"
+  - name: Create an Openshift cluster with Log Analytics monitoring enabled
+    text: az openshift create -g MyResourceGroup -n MyManagedCluster --workspace-resource-id {WORKSPACE_ID}
 """
```

Comment:
Azure Log Analytics Workspace has both Id (i.e. GUID) and Resource Id, I think, we should rename to "WORKSPACE_ID" to "WORKSPACE_RESOURCE_ID" . Since we have seen customers passing WorkspaceID (i.e. GUID) instead of Azure Resource Id of the Log analytics workspace.

Reply:
I agree that changing this to `{WORKSPACE_RESOURCE_ID}` makes sense.  Change submitted in commit c5fe9dcc